### PR TITLE
chore :update logo and social icons in readme

### PR DESCRIPTION
**Pull request by Ayush7614**
This PR Contains
* update logo and social icons in readme
* Added all social icons in Readme
* Update some info

### CODE REVIEW

**Comment by Ayush7614** [comment]
cc @nehagup

**Comment by Ayush7614** [comment]
CC @nehagup

File: README.md [hunk 1]
```diff
@@ -1,11 +1,21 @@
+# Welcome to Keploy 👋
+
+<p style="text-align:center;" align="center">
+  <img align="center" src="https://github.com/keploy/keploy/tree/main/src/images/logo.png" width="30%" />
```
Comment (nehagup) [inline README.md:4]: It would be awesome to use the image from Keploy org profile page.

File: README.md [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
+</p>
+
+[![Slack](https://img.shields.io/badge/slack-@keyploy--slack-yellow.svg)](https://join.slack.com/t/keploy/shared_invite/zt-12rfbvc01-o54cOG0X1G6eVJTuI_orSA)
```
Comment (nehagup) [inline README.md:7]: This is already present. Is this different from the existing one?

File: README.md [hunk 3]
```diff
@@ -1,11 +1,21 @@
...
+[![Github](https://img.shields.io/badge/github-@keyploy--github-9cf.svg)](https://github.com/keploy)
```
Comment (nehagup) [inline README.md:8]: Since this is the main Keploy Repository, it is not required to mention the keploy org profile here.

File: README.md [hunk 4]
```diff
@@ -1,11 +1,21 @@
...
+[![LinkedIN](https://img.shields.io/badge/linkedin-@keyploy--LinkedIn-lightgray.svg)](https://www.linkedin.com/company/keploy/)
```
Comment (nehagup) [inline README.md:9]: Please add LinkedIn, YT, all other new badges at the end of the series of the existing ones.

File: README.md [hunk 5]
```diff
@@ -1,11 +1,21 @@
...
+[![YouTube](https://img.shields.io/badge/youtube-@keyploy--youtube-red.svg)](https://www.youtube.com/channel/UC6OTg7F4o0WkmNtSoob34lg)
 [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen?logo=github)](CODE_OF_CONDUCT.md) 
 [![Tests](https://github.com/keploy/keploy/actions/workflows/go.yml/badge.svg)](https://github.com/keploy/keploy/actions)
 [![Go Report Card](https://goreportcard.com/badge/github.com/keploy/keploy)](https://goreportcard.com/report/github.com/keploy/keploy)
 [![Coverage Status](https://coveralls.io/repos/github/keploy/keploy/badge.svg?branch=main)](https://coveralls.io/github/keploy/keploy?branch=main)
 [![Slack](.github/slack.svg)](https://join.slack.com/t/keploy/shared_invite/zt-12rfbvc01-o54cOG0X1G6eVJTuI_orSA)
 [![License](.github/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-# Keploy
+# About Us
```
Comment (nehagup) [inline README.md:18]: IMO "Keploy" would make more sense for the definition provided in the following paragraph. "About us" section could be added in the documentation with more details around the motivation for Keploy, the team etc.

**Review by nehagup — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Ayush7614) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (Ayush7614) [reply README.md]: Change Done

File: README.md (hunk 3 above)
Comment (Ayush7614) [reply README.md]: Remove

File: README.md (hunk 4 above)
Comment (Ayush7614) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (Ayush7614) [reply README.md]: Done

**Comment by Ayush7614** [comment]
cc @nehagup

**Review by nehagup — CHANGES_REQUESTED** [review]
@Ayush7614  These are last 2 changes required. Then we're set to merge this. :) 


1. As discussed on other PRs, we can directly pick the image from Keploy org page to keep it centralised. https://avatars.githubusercontent.com/u/92252339?s=200&v=4

Please remove this file -> src/images/logo.png

2. Typo in the name of Keploy. 

Please make sure you pull the latest changes before pushing final changes. :)

**Comment by Ayush7614** [comment]
> @Ayush7614 These are last 2 changes required. Then we're set to merge this. :)
> 
> 1. As discussed on other PRs, we can directly pick the image from Keploy org page to keep it centralised. https://avatars.githubusercontent.com/u/92252339?s=200&v=4
> 
> Please remove this file -> src/images/logo.png
> 
> 2. Typo in the name of Keploy in the Readme line 15.
> 
> Please make sure you pull the latest changes before pushing final changes. :)

cc @nehagup

**Comment by nehagup** [comment]
@Ayush7614 It looks like you need to take a pull for the latest changes. 

Also, since the logo.png is not used now, we can remove that file. :)

**Review by nehagup — APPROVED** [review]

**Pull request merged**